plugins/intercom/index.js

```diff
@@ -0,0 +1,9 @@
+import SDK from "./SDK";
+
+module.exports = function(PluginSDK) {
```

Comment:
What problem does this solve?

Reply:
This is the common interface of the current plugin system. This is how we ensure we use the very same instance of `SDK` for all plugins.

Reply:
How does it ensure that?